Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.Properties.cs

```diff
@@ -111,6 +116,15 @@ public double CompactPaneLength
         /// <summary>
         /// Gets or sets the Brush to apply to the background of the Pane area of the control.
         /// </summary>
+        public Brush AccentColor
```

Comment:
Instead of a property on the control itself, define a brush resource that is used for the background of the button. Based on your examples, you would want two brushes, one for the hamburger background and one for the selection brush. 
Example:
<SolidColorBrush x:Key="HambugerMenuHamburgerBackgroundBrush" Color="#FFFFFF"/>
<SolidColorBrush x:Key="HambugerMenuSelectionHighBrush" Color="#FFFFFF"/>